docs/source/dev_guide/testing.rst

```diff
@@ -37,6 +38,19 @@ Before running ``tests/integration/test_complete_run.py`` run the following:
        rm -rf /lcrc/group/e3sm/ac.forsyth2/zppy_test_complete_run_output/v2.LR.historical_0201/post
        zppy -c tests/integration/test_complete_run.cfg
 
+Before running ``tests/integration/test_bundles.py`` run the following:
+
+    .. code::
+
+       rm -rf /lcrc/group/e3sm/public_html/diagnostic_output/ac.forsyth2/zppy_test_bundles_www/v2.LR.historical_0201
+       rm -rf /lcrc/group/e3sm/ac.forsyth2/zppy_test_bundles_output/v2.LR.historical_0201/post
+       zppy -c tests/integration/test_bundles.cfg
+       # bundle1 and bundle2 should run. After they finish, run:
+       rm /lcrc/group/e3sm/ac.forsyth2/zppy_test_bundles_output/v2.LR.historical_0201/post/scripts/bundle3.bash
```

Comment:
@golaz a few options included in the new second commit:

Simply remove the `FileExistsError`: tests still pass except for the bash file content test. Basically, if you rerun `zppy` while `bundle1` is running, `bundle1.bash` gets overwritten (so it doesn't have the lines to run scripts that already have status files with "OK"). Presumably work begins twice on scripts that have not yet been run.

**Check for existence of a status file (e.g., `bundle1.status`) rather than a bash file: this seems to be the best solution.** 
- Downside is if `bundle1` is still running but has finished the task that `bundle3` needs, `zppy` will error out before having a chance to run `bundle3` (it will try to re-run `bundle1`, which is currently running, before getting to `bundle3`).
- Another downside is that it takes a few seconds for the status files to get created, so it's still possible to launch two identical jobs if you re-run `zppy` fast enough.
- Upside is that there is no need to delete `bundle3.bash` or any other files anymore -- `bundle3.status` doesn't get created until after it actually starts running.
- Tests still pass with this change, aside from the bash file content test (will need to include the line printing to the bundle's status file).

Lastly, I included, in a commented out section, code to print an error status if a particular script fails: while this would allow us to have a status simply beyond "the bundle has started", it does add of a lot lines to the scripts, for not much added value, I think.